nautobot/dcim/filters.py

```diff
@@ -878,13 +878,28 @@ class InterfaceFilterSet(
         field_name="pk",
         label="Device (ID)",
     )
+    device_with_common_vc = MultiValueUUIDFilter(
+        method="filter_device_common_vc_id",
+        field_name="pk",
+        label="Device (ID)",
+    )
```

Comment:
This breaks down because of the way that `Device.vc_interfaces` (and the new `Device.common_vc_interfaces`) work, because they fail open to return all interfaces on the device if NOT a member of a VC. Therefore all the code paths retrieving interfaces just use these methods with the expectation that if NOT a VC member, you'll still get the interfaces.